backend/typescript/services/implementations/campService.ts

```diff
@@ -1481,6 +1541,25 @@ class CampService implements ICampService {
     }
   }
 
+  async deleteFormQuestionsByIds(
+    formQuestionIds: string[],
+    dbSession?: mongoose.ClientSession,
```

Comment:
sidenote, are helper methods supposed to be declared in the interface?

Reply:
hmm I am actually not sure. Should I remove them?

Reply:
No idea, I'm happy to just keep it